tests/conftest.py

```diff
@@ -67,6 +68,10 @@ def provider():
         'authlib_args': {'client_id': 'test', 'client_secret': 'test'},
         'cache': MemoryCache
     }
+
+    # hasten request retries for tests
+    flask_multipass_cern.retry_config.max_retries.backoff_factor = 0
```

Comment:
it'd be cleaner to use the `monkeypatch` fixture for this so it's only set while this fixture is active. doesn't really make a difference though since leaving this value at 0 won't do any harm